test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/Helpers/BaseAcceptance.cs

```diff
@@ -10,11 +14,81 @@ namespace Microsoft.Testing.Platform.Acceptance.IntegrationTests;
 /// </summary>
 public abstract class BaseAcceptanceTests : TestBase
 {
+    internal static string RID { get; private set; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "win-x64" : "linux-x64";
+
+    public static string MSTestCurrentVersion { get; private set; }
+
+    static BaseAcceptanceTests()
+    {
+        foreach (var package in Directory.GetFiles(Constants.ArtifactsPackagesNonShipping, "*MSTest*.nupkg", SearchOption.AllDirectories)
```

Comment:
Fine as-is but do we need the extra logic? I mean can't we pick one specific package (e.g. `MSTest.TestFramework`) and not look at the rest?

Reply:
This also gives the benefit of not needing nuget API, we can just substring the package name to get the version.

Reply:
> Fine as-is but do we need the extra logic? I mean can't we pick one specific package (e.g. MSTest.TestFramework) and not look at the rest?

It was on purpose if you've more than one packages version produced and restore could pick up weird one with warning but run tests without problem, at least in local happened to me.

Reply:
removed